polyfill/lib/calendar.mjs

```diff
@@ -1414,7 +1419,7 @@ function isGregorianLeapYear(year) {
 /** Base for all Gregorian-like calendars. */
 const makeHelperGregorian = (id, originalEras) => {
   const { eras, anchorEra } = adjustEras(originalEras);
-  return ObjectAssign({}, nonIsoHelperBase, {
+  const helperGregorian = ObjectAssign({}, nonIsoHelperBase, {
```

Comment:
What's this change for?

Reply:
This one is pure refactor, nothing functional. Instead of returning the object literal, it assigns a local variable to the object literal, then returns that variable in the line immediately after the literal definition block. This is an optional change on the JS side (It was helpful for typing on the TS side) and I 'd be happy to revert it if you'd prefer.

BTW the way I made this PR was to compile the current TS down to JS, diff it with the current JS implementation, and then reverted most of the differences, leaving only those that were legit bug fixes, which seemed to help readability, or that would make it easier to exchange fixes between TS<->JS polyfills in the future. The top priority was to make sure that there wasn't unexpected divergence e.g. bugs fixed on one side but not the other. I found a few of those. 😄

Reply:
I removed this change-- not important enough.